### ISSUE TITLE
2021-11-11

### ISSUE THREAD

**Issue by ssfc** [ssfc/wangdao#652]
3312.Figure 2: (1) Two separate data augmentation operators; (2) A base encoder network f(); (3) a projection head g() are trained to maximize agreement using a contrastive loss; (4) throw away the projection head g() and use encoder f(); (2021-10-26)
3323.(1) positive pair; random cropping, random color distortions, and random Gaussian blur; (2021-10-26)
3334.(2) base encoder f() that extracts representation vectors from augmented data examples; adopt the commonly used ResNet; (2021-10-26)
3345.(3) A small neural network projection head g() that maps representations to the space where contrastive loss is applied; (2021-10-26)
3356.(4) A contrastive loss function defined for a contrastive prediction task; (2021-10-26)
3367.Treat the other 2(N - 1) augmented examples within a minibatch as negative examples; NT-Xent (the normalized temperature-scaled cross entropy loss); (2021-10-26)
337II.2. Training with Large Batch Size
3381.do not train the model with a memory bank; (2021-10-26)
3392.train our model with Cloud TPU; (2021-10-26)
